packages/cli/api-importers/openapi/openapi-v2-importer/src/3.1/schema/SchemaConverter.ts

```diff
@@ -43,7 +43,6 @@ export class SchemaConverter extends AbstractConverter<OpenAPIConverterContext3_
         context: OpenAPIConverterContext3_1;
         errorCollector: ErrorCollector;
     }): SchemaConverter.Output | undefined {
-        // Try to convert as enum
```

Comment:
thank you for deleting these -- i think being forced to add comments is usually a indication that the code isn't easy to read so lets aim for reducing comments!